Stylesheets/PageStyles/authentication.css

```diff
@@ -1,13 +1,64 @@
+
 .main {
-    padding-top: 10rem;
+    grid-template-columns: auto;
+   
+}
+
+.background {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    filter: blur(1px);
+}
+
+.product-gryffindor {
+    background-color: #800000;
+    width: 100%;
+    height: 50vh;
```

Comment:
You are repeating the styles here, instead you can make a separate class and add it there

Reply:
Okay yes I am changing this too